lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -102,7 +102,8 @@ sub _where_filter
 
     if (defined $filter) {
         if (exists $filter->{name}) {
-            push @query, "(to_tsvector('mb_simple', release.name) @@ plainto_tsquery('mb_simple', ?) OR release.name = ?)";
+            push @query,
```

Comment:
Why the extra push here?

Reply:
Looks like a rebase issue. Fixed, thanks!